Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,1249 @@
+            outputs |= {'Id': extracted_issue_id, 'Key': issue_id_or_key}
+        human_readable = tableToMarkdown("Comments", comments)
+        return CommandResults(
+            outputs_prefix='Ticket',
+            outputs=outputs,
+            outputs_key_field='Id',
+            readable_output=human_readable,
+            raw_response=res
+        )
+        # {'Ticket((val.Id && val.Id == obj.Id) || (val.Key && val.Key == obj.Key))': outputs},
+    else:
+        return CommandResults(readable_output='No comments were found in the ticket')
+
+
+def get_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> List[CommandResults]:
+    issue_id_or_key = args.get('issue_id', '') if args.get('issue_id', '') else args.get('issue_key', '')
+    if not issue_id_or_key:
+        return_error(ID_OR_KEY_MISSING_ERROR)
+    headers = args.get('headers', '')
+    get_attachments = argToBoolean(args.get('get_attachments', False))
+    expand_links = argToBoolean(args.get('expand_links', False))
+    responses: List[Dict[str, Any]] = []
+    res = client.get_issue(issue_id_or_key=issue_id_or_key)
+    responses.append(res)
+    command_results: List[CommandResults] = []
+    if(expand_links):
+        for sub_task in res.get('fields', {}).get('subtasks', []):
+            responses.append(client.get_issue(full_issue_url=sub_task.get('self', '')))
+        for link_issue in res.get('fields', {}).get('issuelinks', []):
+            responses.append(client.get_issue(full_issue_url=link_issue.get('inwardIssue', {}).get('self', '')))
+    for response in responses:
+        markdown_dict, outputs = create_issue_md_and_outputs(client=client, res=response, get_attachments=get_attachments)
+        command_results.append(
+            CommandResults(
+                outputs_prefix='Ticket',
+                outputs=outputs,
+                outputs_key_field='Id',
+                readable_output=tableToMarkdown(name=f'Issue {outputs.get("Key", "")}', t=markdown_dict,
+                                                headers=argToList(headers),
+                                                headerTransform=pascalToSpace),
+                raw_response=response
+            ))
+    return command_results
+
+
+def create_issue_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    issue_args: Dict[str, Any] = args
+    epic_name = args.get('epic_name', '')
+    issue_args['labels'] = issue_args.get('labels', '').split(',')
+    issue_args['components'] = [{"name": component} for component in argToList(issue_args.get('components'))]
+    issue_fields = create_issue_fields(issue_args=issue_args, issue_fields_mapper=client.ISSUE_FIELDS_MAPPER)
+
+    issue_fields.get('fields', {})['description'] = text_to_adf(text=args.get('description', ''))
+    issue_fields.get('fields', {})['environment'] = text_to_adf(text=args.get('environment', ''))
+    if epic_name:
+        issue_fields.get('fields', {})['customfield_10010'] = epic_name
```

Comment:
I talked to the TPM and he suggested to not offer this functionality (creating issues of type Epic), since this will require the user to enter the id of the customfield, and we don't want that, if the user wants to create an issue of type epic, then they can use the issue_json.